### ISSUE TITLE
Issue with v 1.0.3

### ISSUE THREAD

**Issue by martinbertinat** [EasyCorp/easy-deploy-bundle#53]
Hello, today I've updated to v1.0.3 and my deploy script is not working.

Im running symfony 3.3, here is my config:

```
public function configure()
    {
        return $this->getConfigBuilder()
            ->server("servername")
            ->deployDir($this->getDeployDir())
            ->repositoryUrl($this->repoUrl)
            ->remoteComposerBinaryPath('composer')
            ->composerInstallFlags('--prefer-dist --no-interaction --quiet')
            ->sharedFilesAndDirs(array('app/config/parameters.yml', 'var/logs', 'var/sessions'))
            ->writableDirs(array('var/cache/', 'var/logs/', 'var/sessions'))
            ;
    }
```

This is the error when I run "deploy stag":

 ```
[InvalidArgumentException]                                                 
                                                                               
    The target directory "/home/user/client_repo/releases/20180404184402/public" does not exist. 
```

I think it's something related to symfony 4 and the "public" directory.

Thanks!

**Comment by The-Don-Himself** [comment]
Yeah getting this too, I think the checks for the Symfony version might be failing and defaulting to 4. Pinging @javiereguiluz incase you haven't seen this, it breaks BC.

**Comment by gabricom** [comment]
Same problem . 
You can fix this temporarily by adding  ->webDir('web') to the deployer

**Comment by The-Don-Himself** [comment]
@gabricom thanks for the suggested fix, I was wondering though, have you tried it and deployed successfully? Because there a other changes in directory structure between Symfony 3.4.x and 4.x.

**Comment by gabricom** [comment]
I tried and it works for me, the other directories seems to be fine

**Comment by peterbutorin** [comment]
It work if you set shared files and dirs. If it is not set - null break execution. And if I don’t add ssh keys from server to gitlab I cannot enter username and password. On local machine I can do this. And if I use .env.dist on local it does not copy by command cp -a ./repo/* to ./releases/release. It is for example. And it don’t copy hidden files. And composer return error. This regular must be not like *. It is on symfony 4+.

Отправлено с iPhone
